tests/smb-nt-create-andx-request-02/test.yaml

```diff
@@ -0,0 +1,17 @@
+# skip as it fails
+skip: true
```

Comment:
I see 4 alerts on this instead of 3. Is that the failure?

Reply:
that's a difference between 4.1.x and 5.0.x although I had 3 on 5.0.1 as well for example. Will try to narrow it down to a commit/version.

Reply:
So the first narrow down shows that 4.1.7 and 5.0.1 behave the same. With 5.0.2 the difference appears with the first test, while the others are still the same. What's a bit strange is this https://github.com/norg/suricata-verify/runs/620367010?check_suite_focus=true also shows the same issue but with the major versions the other way round.

So this made me testing more and it seems to be another issue. It can happen with each of those version (4.1.7, 5.0.1, 5.0.2). I did run 100 tests with `run.py smb-nt-create-andx-request-01` on each of those and it happened more often on 5.0.2 and less often on 4.1.7.

I did test the other ones in that commit as well but each had the same amount of events in those runs. So it's only a strange behavior in `smb-nt-create-andx-request-01`.